src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -4768,6 +4800,46 @@ internal static Collection<string> GetResolvedPathCollection(string filePath, Ex
             return filePaths;
         }
 
+        internal void CreateWindowsPowerShellCompatResources()
+        {
+            lock(WindowsPowerShellCompatSyncObject)
+            {
+                if (WindowsPowerShellCompatRemotingSession == null)
+                {
+                    var commandInfo = new CmdletInfo("New-PSSession", typeof(NewPSSessionCommand));
+                    var ps = System.Management.Automation.PowerShell.Create(RunspaceMode.CurrentRunspace)
+                        .AddCommand(commandInfo)
+                        .AddParameter("UseWindowsPowerShell", true)
+                        .AddParameter("Name", WindowsPowerShellCompatRemotingSessionName);
+
+                    var results = ps.Invoke<PSSession>();
+                    if (results.Count > 0)
+                    {
+                        WindowsPowerShellCompatRemotingSession = results[0];
+
+                        var sb = ScriptBlock.Create(string.Format("Remove-PSSession -Name {0}", WindowsPowerShellCompatRemotingSessionName));
+                        Events.SubscribeEvent(null, null, PSEngineEvent.Exiting, null, sb, false, false);
```

Comment:
Please use named arguments.

Reply:
Yes, that would make it easier to read.
This cleanup code was removed as unnecessary based on feedback.